core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -239,12 +269,15 @@ private[spark] class SparkDateTimeException(
 private[spark] class SparkFileAlreadyExistsException(
     errorClass: String,
     errorSubClass: Option[String] = None,
-    messageParameters: Array[String])
+    messageParameters: Map[String, String])
```

Comment:
The exception is used only once by the error class FAILED_RENAME_PATH but it is not tested at all. There is the JIRA for testing https://issues.apache.org/jira/browse/SPARK-38728